lib/task_bunny/failure_backend/logger.ex

```diff
@@ -0,0 +1,99 @@
+defmodule TaskBunny.FailureBackend.Logger do
+  @moduledoc """
+  Default failure backend that reports job errors to Logger.
+  """
+  use TaskBunny.FailureBackend
+  require Logger
+  alias TaskBunny.JobError
+
+  def report_job_error(error = %JobError{error_type: :exception}) do
+    message = """
+    TaskBunny - #{error.job} failed for an exception.
+
+    Exception:
+    #{my_inspect error.exception}
+
+    #{common_message error}
+
+    Stacktrace:
+    #{Exception.format_stacktrace(error.stacktrace)}
+    """
+
+    do_report(message, error.reject)
+  end
+
+  def report_job_error(error = %JobError{error_type: :return_value}) do
+    message = """
+    TaskBunny - #{error.job} failed for an invalid return value.
+
+    Return value:
+    #{my_inspect error.return_value}
+
+    #{common_message error}
+    """
+
+    do_report(message, error.reject)
```

Comment:
More an option suggestion, I know your preference, so I'm just putting it out there, because it will align the triple-quotes nicely.

If you like it you can also write:

```
"""
TaskBunny - #{error.job} failed for an invalid return value.

Return value:
#{my_inspect error.return_value}

#{common_message error}
"""
|> do_report(error.reject)
```

(You know how I love aligning and pipes 😛)